src/modules/Dropdown/Dropdown.js

```diff
@@ -798,6 +798,7 @@ export default class Dropdown extends Component {
         active={isActive(opt.value)}
         onClick={this.handleItemClick}
         selected={selectedIndex === i}
+        onMouseDown={e => e.preventDefault()}
```

Comment:
Let's add a comment here something to effect of:

```js
// prevent default to allow item select without closing on blur
```

This way, looking at this line of code later on gives us some inclination as to why we are preventing default.